src/Modules/UngradedModules.jl

```diff
@@ -7,6 +7,7 @@ export presentation, coords, coeffs, repres, cokernel, index_of_gen, sub,
       free_module, tor, lift_homomorphism_contravariant, lift_homomorphism_covariant, 
       hom_without_reversing_direction, ext, map_canonically, 
       all_canonical_maps, register_morphism!, dense_row, 
+      show_subquo, show_morphism, show_morphism_as_map,
```

Comment:
What is the point of these methods?

Reply:
These are alternative show functions which display additional information compared to our standard choice of show functions. The information is very useful in specific applications. The functions will be documented.